src/Database/Query.php

```diff
@@ -280,9 +238,9 @@ public function primaryKeyName(string $primaryKeyName)
 	 * By default all columns will be selected
 	 *
 	 * @param mixed $select Pass either a string of columns or an array
-	 * @return \Kirby\Database\Query
+	 * @return $this
 	 */
-	public function select($select)
+	public function select(mixed $select): static
```

Comment:
`$this->select` property type is `array|string|null` but this method accept as `mixed`? Not consistent.